Tasks/AzureTestPlanV0/Invokers/goinvoker.ts

```diff
@@ -1,15 +1,30 @@
 import tl = require('azure-pipelines-task-lib/task');
 import utils = require('../utils');
 import constants = require('../constants');
+import { executeGo, executeGotestsum } from '../testexecutor';
 
-export async function executeGoTests(testsToBeExecuted: string[]): Promise<number> {
+//GO command like "gotestsum --junitfile TEST-Go<i>-junit.xml -- <filepath> -v -run ^<TestName>$"
+export async function executeGoTests
+    (testsToBeExecuted: string[]): Promise<number> {
 
-    //Go execution will be added
-    /*executable = go;
-    args = test, ./...  
-    spawn*/
-
-    console.log("Go changes1");  
-    return 1;
-}
+    let command = constants.INSTALL;
+    let argument = constants.GOTESTSUM_PACKAGE;
+    const installGotestsum = await executeGo(command, argument);
+    console.log(installGotestsum);
 
+    let i = 0;
+    for (let tests of testsToBeExecuted) {
+        try {
+            const modifiedPath = utils.separatePath(tests);
+            const modifiedTest = utils.separateTestName(tests);
+            const command = constants.GOTESTSUM_JUNITFILE;
```

Comment:
Shouldn't this be `constants.GOTESTSUM_PACKAGE` ?

Reply:
GOTESTSUM_PACKAGE = 'gotest.tools/gotestsum@latest'; 
//go install gotest.tools/gotestsum@latest'
GOTESTSUM_JUNITFILE = '--junitfile';
//GO command like "gotestsum --junitfile TEST-Go<i>-junit.xml -- <filepath> -v -run ^<TestName>$"
No, GOTESTSUM_PACKAGE is different used for installing gotestsum and GOTESTSUM_JUNITFILE is a part of command to create junit file.

Reply:
Added